### PR TITLE
Correctly process tilde sign in composes url

### PR DESCRIPTION
**Pull request by vslinko**

### CODE REVIEW

**Comment by vslinko** [comment]
Fixes https://github.com/webpack/css-loader/issues/254

File: lib/processCss.js [hunk 1]
```diff
@@ -67,6 +67,9 @@ var parserPlugin = postcss.plugin("css-loader-parser", function(options) {
 			} else if(/^:import\(.+\)$/.test(rule.selector)) {
 				var match = /^:import\((.+)\)$/.exec(rule.selector);
 				var url = loaderUtils.parseString(match[1]);
+				if(loaderUtils.isUrlRequest(url, options.root) && options.mode === "global") {
+					url = loaderUtils.urlToRequest(url, options.root);
+				}
```
Comment (vslinko) [inline lib/processCss.js]: Copy-paste from `@import` processing function. Please review this code.

**Comment by vslinko** [comment]
Looks like a problem with github-codecov integration.

**Comment by vslinko** [comment]
I don't know what I'm fixing here because I found a real reason of #254

**Comment by sokra** [comment]
This PR is not needed. composes use always module style imports:

`composes: a from "b/x"` => `require("a/x").a`

**Pull request closed without merging**